canine/orchestrator.py

```diff
@@ -260,7 +278,7 @@ def run_pipeline(self, output_dir: str = 'canine_output', dry_run: bool = False)
                 runtime/3600,
```

Comment:
Need to update `estimate_cost` functions to work with seconds; I've left this /3600 as a reminder.

Reply:
When you do, do you mind updating it for all backends?

Reply:
Absolutely!